examples/chip-tool-darwin/templates/tests.js

```diff
@@ -248,18 +248,12 @@ function getTests()
     // some failures on darwin supposely because the ReadClient stays open for the
     // whole duration of the tests and that goes past some internal limits.
     // Because of this, some of the tests are disabled on darwin.
-    'Test_TC_WNCV_1_1',
-    'Test_TC_WNCV_2_1',
-    'Test_TC_WNCV_2_2',
-    'Test_TC_WNCV_2_4',
-    'Test_TC_WNCV_2_5',
+    'Test_TC_WNCV_1_1', 'Test_TC_WNCV_2_1', 'Test_TC_WNCV_2_2', 'Test_TC_WNCV_2_4', 'Test_TC_WNCV_2_5',
     //'Test_TC_WNCV_3_1',
     //'Test_TC_WNCV_3_2',
     //'Test_TC_WNCV_3_3',
-    'Test_TC_WNCV_3_4',
-    'Test_TC_WNCV_3_5',
-    'Test_TC_WNCV_4_3',
-    'Test_TC_WNCV_4_4',
+    'Test_TC_WNCV_3_4', 'Test_TC_WNCV_3_5', 'Test_TC_WNCV_4_3', 'Test_TC_WNCV_4_4',
```

Comment:
Please leave one per line.

Reply:
Yep not me, the automated formatter did it....
How to avoid this ?